test/integrationtest.py

```diff
@@ -487,28 +489,25 @@ def run():
 
     # Add a test method for the configuration test for each language.
     if "parser" in selectedTests:
-        map(IntegrationTest.addParserTest, selectedLanguages)
+        list(map(IntegrationTest.addParserTest, selectedLanguages))
```

Comment:
When is `map` permitted? Previous uses where changed to `[… for … in …]`.

Reply:
people usually prefer list comprehension, but map or filter etc are not strictly forbidden